core/templates/domain/feedback_message/FeedbackMessageSummaryObjectFactory.ts

```diff
@@ -54,9 +55,11 @@ export class FeedbackMessageSummaryObjectFactory {
   createNewMessage(
       newMessageId: number, newMessageText: string, authorUsername: string,
       authorPictureDataUrl: string): FeedbackMessageSummary {
+    // Date.now() returns number of milliseconds since 1970-01-01 UTC.
+    let createdOn: number = new Date().getTime();
     return new FeedbackMessageSummary(
       newMessageId, newMessageText, null, null, null, null, authorUsername,
-      authorPictureDataUrl, new Date());
+      authorPictureDataUrl, createdOn);
```

Comment:
createdOnMsecs? (Dimensional numbers should always have units in the name.)

Reply:
Done.